scripts/globals/items/death_scythe.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------------
+-- ID: 16777
+-- Item: Death Scythe
+-- Additional Effect: Drains HP
+-----------------------------------------
+require("scripts/globals/status");
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(player,target,damage)
+    local chance = 10;
+    if (target:getMainLvl() > player:getMainLvl()) then

```

Comment:
NO. I am sick of explaining these are NOT level corrected and that even if it were this method of attempting level correction isn't even anywhere near accurate.